simulator/src/test/java/com/hazelcast/simulator/probes/impl/HdrProbeTest.java

```diff
@@ -26,16 +30,19 @@ public void testConstructor_noThroughputProbe() {
         assertFalse(tmpProbe.isPartOfTotalThroughput());
     }
 
+    // this test is fragile and can easily fail if there is some kind of pausing going on
```

Comment:
Why did you replace my "check the latency range" with this more fragile solution? The existing range check didn't fail in ages.